openapi-cli/src/test/resources/ballerina-to-openapi/expected_gen/constraint/decimalMax.yaml

```
@@ -0,0 +1,64 @@
+openapi: 3.0.1
+info:
+  title: PayloadV
+  version: 0.0.0
+servers:
+  - url: "{server}:{port}/payloadV"
+    variables:
+      server:
+        default: http://localhost
+      port:
+        default: "9090"
+paths:
+  /pet:
+    post:
+      operationId: postPet
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/School'
+      responses:
+        "500":
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorPayload'
+components:
+  schemas:
+    School:
+      required:
+        - marks
+      type: object
+      properties:
+        marks:
+          $ref: '#/components/schemas/Marks'
+    ErrorPayload:
+      type: object
+      properties:
+        reason:
+          type: string
+          description: Reason phrase
+        path:
+          type: string
+          description: Request path
+        method:
+          type: string
+          description: Method type of the request
+        message:
+          type: string
+          description: Error message
+        timestamp:
+          type: string
+          description: Timestamp of the error
+        status:
+          type: integer
+          description: Relevant HTTP status code
+          format: int32
+    Marks:
+      maximum: 5.55
+      exclusiveMaximum: true
+      minimum: 2.55
+      type: number
+      format: double
```

Add new line at the end of the file.
```suggestion
      format: double

```